__tests__/setup/createUserId.js

```diff
@@ -0,0 +1,3 @@
+export default function createUserId() {
+  return `u-${ Math.random().toString(36).substr(2, 5) }`;
```

Comment:
Should we use 'dl_' instead of 'u'? Will the streaming extension work/require ids to start with 'dl_'?